lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -1,19 +1,17 @@
---extra-index-url https://wheels.galaxyproject.org/simple
```

Comment:
Was that intentional ?

Reply:
Not sure what changed, but it was removed automatically when doing `poetry export ...` . I don't think it matters as we explicitly pass the `--extra-index-url` option to pip in `scripts/common_startup.sh` (and all tests still pass).